ui-components/src/blocks/ImageCard.tsx

```diff
@@ -20,27 +20,26 @@ export interface ImageTag {
   text?: string
   iconType?: UniversalIconType
   iconColor?: string
+  styleType?: AppearanceStyleType
 }
 
 export interface ImageCardProps {
+  /** A description of the image, used as alt text */
```

Comment:
I like these comments! Is this something we do when defining props everywhere?

Reply:
Yeah we experimented with a couple of different ways of defining prop comments, and this one (putting them above each one in the TS interface) seems to work best for showing up in the Storybook docs tab.